probe-rs/src/debug/debug_info.rs

```diff
@@ -95,6 +96,8 @@ impl DebugInfo {
 
         while let Ok(Some(header)) = iter.next() {
             if let Ok(unit) = dwarf_cow.unit(header) {
+                // TODO: maybe it's not correct to read from arbitrary units
```

Comment:
This is probably just a coincidence and not a general truth. Still, this was the easiest 4-byte pointer size I found nearby.